README.md
# TEMPLATE
## INI UNTUK TEMPLATE PEKERJAAN FREELANCER
### Silahkan Pilih View Seperti apa yang anda inginkan

- [TEMPLATE 1](https://lgarin211.github.io/TEMPLATE/Finance1/index.html)
- [TEMPLATE 2](https://lgarin211.github.io/TEMPLATE/Presento/index.html)
- [TEMPLATE 3](https://lgarin211.github.io/TEMPLATE/Tenpurēto/index.html)




Presento/changelog.txt
Version: 3.1.0
  - Updated Bootstrap to version 5.0.0-beta3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to V3.1

Version: 3.0.1
  - Updated Bootstrap to version 5.0.0-beta2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 3.0.0
  - The template does not require jQuery anymore
  - Removed jQuery and all the jQuery plugins
  - The assets/js/main.js was rewritten completely with vanilla Javascript. No more jQuery code
  - Restructured the dev version for better development experience
  - Updated the PHP Email Form to V3.0 - No jQuery dependency. Added attachment support

Version: 2.0.3
  - Updated Bootstrap to version 5.0.0-beta1
  - Updated the PHP Email Form to v2.3
  - Other small fixes and improvements
  
Version: 2.0.2
  - Updated Bootstrap to version 5.0.0-alpha3
  - Updated all outdated third party vendor libraries to their latest versions

Version: 2.0.1
  - Update Bootstrap v5.0 to Alpha 2
  - Updated all outdated third party vendor libraries to their latest versions
  
Version: 2.0.0
  - Initial release with using the Bootstrap v5.0 (Alpha)
  
Version: 1.0.0
  - Initial Release


Tenpurēto/assets/js/main.js
(function ($) {

    "use strict";

    $(window).on('load', function () {


        /*Page Loader active
        ========================================================*/
        $('#preloader').fadeOut();

        // Sticky Nav
        $(window).on('scroll', function () {
            if ($(window).scrollTop() > 50) {
                $('.scrolling-navbar').addClass('top-nav-collapse');
            } else {
                $('.scrolling-navbar').removeClass('top-nav-collapse');
            }
        });

        // one page navigation 
        $('.navbar-nav').onePageNav({
            currentClass: 'active'
        });

        /* slicknav mobile menu active  */
        $('.mobile-menu').slicknav({
            prependTo: '.navbar-header',
            parentTag: 'liner',
            allowParentLinks: true,
            duplicate: true,
            label: '',
            closedSymbol: '<i class="lni-chevron-right"></i>',
            openedSymbol: '<i class="lni-chevron-down"></i>',
        });

        /* WOW Scroll Spy
      ========================================================*/
        var wow = new WOW({
            //disabled for mobile
            mobile: false
        });

        wow.init();

        /* Testimonials Carousel 
       ========================================================*/
        var owl = $("#testimonials");
        owl.owlCarousel({
            loop: true,
            nav: false,
            dots: false,
            center: true,
            margin: 15,
            slideSpeed: 1000,
            stopOnHover: true,
            autoPlay: true,
            responsiveClass: true,
            responsiveRefreshRate: true,
            responsive: {
                0: {
                    items: 1
                },
                768: {
                    items: 1
                },
                960: {
                    items: 1
                },
                1200: {
                    items: 1
                },
                1920: {
                    items: 1
                }
            }
        });

        /* Back Top Link active
        ========================================================*/
        var offset = 200;
        var duration = 500;
        $(window).scroll(function () {
            if ($(this).scrollTop() > offset) {
                $('.back-to-top').fadeIn(400);
            } else {
                $('.back-to-top').fadeOut(400);
            }
        });

        $('.back-to-top').on('click', function (event) {
            event.preventDefault();
            $('html, body').animate({
                scrollTop: 0
            }, 600);
            return false;
        });
    });

}(jQuery));

Tenpurēto/assets/js/particlesjs.js
particlesJS("particles-js", {
	"particles": {
		"number": {
			"value": 50,
			"density": {
				"enable": true,
				"value_area": 700
			}
		},
		"color": {
			"value": ["#fc3c5f", "#993cfc", "#3ca9fc", "#3cfc5f", "#fcdf3c"]
		},
		"shape": {
			"type": "circle",
			"stroke": {
				"width": 0,
				"color": "#000000"
			},
			"polygon": {
				"nb_sides": 30
			}
		},
		"opacity": {
			"value": 0.5,
			"random": false,
			"anim": {
				"enable": false,
				"speed": 1.2,
				"opacity_min": 0.15,
				"sync": false
			}
		},
		"size": {
			"value": 2.5,
			"random": false,
			"anim": {
				"enable": true,
				"speed": 2,
				"size_min": 0.15,
				"sync": false
			}
		},
		"line_linked": {
			"enable": true,
			"distance": 80,
			"color": "#d6d6d6",
			"opacity": 1,
			"width": 1
		},
		"move": {
			"enable": true,
			"speed": 1.6,
			"direction": "none",
			"random": false,
			"straight": false,
			"out_mode": "out",
			"bounce": false,
			"attract": {
				"enable": false,
				"rotateX": 600,
				"rotateY": 1200
			}
		}
	},
	"interactivity": {
		"detect_on": "canvas",
		"events": {
			"onhover": {
				"enable": false,
				"mode": "repulse"
			},
			"onclick": {
				"enable": false,
				"mode": "push"
			},
			"resize": true
		},
		"modes": {
			"grab": {
				"distance": 400,
				"line_linked": {
					"opacity": 1
				}
			},
			"bubble": {
				"distance": 400,
				"size": 50,
				"duration": 2,
				"opacity": 8,
				"speed": 3
			},
			"repulse": {
				"distance": 200,
				"duration": 0.4
			},
			"push": {
				"particles_nb": 4
			},
			"remove": {
				"particles_nb": 2
			}
		}
	},
	"retina_detect": true
});